### ISSUE TITLE
Remove outdated/duplicative WordPress migration content

### ISSUE THREAD

**Issue by stevector** [pantheon-systems/documentation#8622]
## Expected behavior

When I google for "Pantheon WordPress migration" I get one reliable entry point as the top result. And lower results fit into a coherent flow of information.

## Actual behavior

Old and duplicative results come up first:

- https://pantheon.io/resources/quickstart-guide-migrating-wordpress-site
- https://docs.pantheon.io/migrate-wordpress

### Steps to reproduce the problem

1. https://www.google.com/search?q=pantheon+wordpress+migration

## Suggested Resolution
* Remove old stuff and redirect to https://pantheon.jp/docs/guides/wordpress-pantheon/migrate-existing-wp-site
* This page should probably be also folded into the general guide: https://pantheon.jp/docs/guides/wpengine/migrate

## Additional information
My suggestion is probably overly simplistic and risks SEO benefits of old content. Please discuss with RJ Horning for a general redirect/SEO perspective. Also, please discuss with @scottbuscemi

**Comment by stevector** [comment]
Reading through more of the existing docs pages in the last (checks timestamp of issue creation) 5 minutes, I'm less confident on my suggested resolution. (Maybe we want people starting at https://pantheon.jp/docs/guides/guided ??) But we can definitely make this info easier to digest.

**Comment by scottbuscemi** [comment]
FWIW your links are pointing out another issue which is `.jp` not redirecting to `.io` ...

**Comment by scottbuscemi** [comment]
I agree that `/guides/wordpress-pantheon/migrate-existing-wp-site` is a great place to point folks to since it covers both manual and automated migrations.

**Comment by kwhitman0** [comment]
I agree with having one focused path. I'd try to make one single page that is the ultimate guide to migrating WordPress, and drive everyone to that one page. You can start with the current page that is linking highest for that term, and then add to it and adjust it, to keep the momentum from search. 

It seems like our focus keyword would be "WordPress migration" so we would want to make sure that is in the URL segment and page title, at the least. Don't worry - you'll still capture people searching for "Pantheon WordPress migration" but we broaden it a bit to bring in other traffic.

@stevector  - To your question about which page to focus on, I would optimize the more general migration page for keyword phrase "Pantheon migration" and link to the WordPress page from it. But I would absolutely keep the WordPress-specific one as a destination as well. Searchers who typed "WordPress" are going to want to see that word immediately on the page, or they are more likely to hit the back button and bounce (and that hurts our ranks). Search engines are a lot more intent based these days, and good at figuring it out, but exact keyword match is still super important for click-through behavior.  

More SEO notes:

- If you redirect any pages, use a 301 redirect. That's Google's [preferred way](https://developers.google.com/search/docs/crawling-indexing/301-redirects) to handle it, so we won't be penalized AND any ranks for that link will just transfer over to the new destination.

- If we have pages with duplicated content for any reason, we can simply add a [canonical tag](https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls#definition). That tells Google which page is the original or preferred link. And then we aren't penalized if we have duplicate content.
